pkgs/development/libraries/opencollada/default.nix

```diff
@@ -21,6 +21,8 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  patches = [ ./pcre.patch ];
+
   patchPhase = lib.optionalString stdenv.isDarwin ''
```

Comment:
@eadwu Change this to postPatch and it should fix macOS building.

Reply:
Oh didn't see patchPhase, just added it to patchPhase